### DISCUSSION TITLE
Multiple FilterFn´s for one column

### DISCUSSION THREAD

**Discussion by asinfdev** [TanStack/table#5187] [General] [answered]
It is currently not possible to set several filter functions to one column. This means that column filters cannot be linked 'and' and 'or'. Can you provide an option for that use case?

**Comment by asinfdev** [comment]
Upvote

**Comment by KevinVandy** [comment, marked as answer]
This is something that should be added in v9

**Reply by asinfdev** [reply]
@KevinVandy thanks for the reply! I'm very pleased to hear that. Is there a rough time frame when V9 will be released? 

Greetings Alex

**Reply by KevinVandy** [reply]
- 3-6 months probably. Not soon. You can still achieve this functionality with manual filtering for now
